scripts/jenkins/cloud/ansible/start-deployer-vm.yml

```diff
@@ -18,7 +18,7 @@
 - name: Start fresh deployer VM
   hosts: "gate-{{ cloud_env }}"
   remote_user: root
-  gather_facts: no
+  gather_facts: yes
```

Comment:
Need to be careful with this one. I ran into trouble in the past where the python version used by Ansible didn't come with the base system.

Reply:
In this case it works however ;-)